symbols/arm9.yml

```diff
@@ -8651,7 +8651,7 @@ arm9:
         r1: divisor
         return: (quotient) | (remainder << 32)
   data:
-    - name: ARM9_HEADER
+    - name: SECURE
```

Comment:
Can we name this more specifically? Something like `SECURE_AREA_HEADER`?

Reply:
Oh I guess this comes from `_secure`, which is some standardized name? If so we can leave it I guess.

Reply:
Yes, I changed the case convention but left it as is otherwise.